src/chat/consumers.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+from channels.generic.websocket import AsyncWebsocketConsumer
+
+
+class ChatConsumer(AsyncWebsocketConsumer):
+    async def connect(self):
+        self.room_name = self.scope["url_route"]["kwargs"]["room_name"]
+        self.room_group_name = "chat_%s" % self.room_name
```

Comment:
Consider using f-strings for string formatting, as they are more readable and efficient than the % operator: ```self.room_group_name = f"chat_{self.room_name}"```